.spdxignore

```diff
@@ -2,8 +2,9 @@ pkg/edgeview/vendor/
 pkg/kube/eve-bridge/vendor/
```

Comment:
Not relevant to this PR, but can we have a more general pattern for ignoring vendor?

Reply:
@OhmSpectator , any comments?

Reply:
Yeah...
In theory, we can put just /vendor/ there... In this case, a file will be ignored if it has "/vendor/" as a part of its path.